genesyscloud/group/resource_genesyscloud_group_schema.go

```diff
@@ -109,9 +109,9 @@ func ResourceGroup() *schema.Resource {
 			"owner_ids": {
 				Description: "IDs of owners of the group.",
 				Type:        schema.TypeList,
-				Required:    true,
+				Optional:    true,
 				Elem:        &schema.Schema{Type: schema.TypeString},
-				MinItems:    1,
+				Computed:    true,
```

Comment:
owner_ids shouldn't have been required. You can create a group without this field, it will just be populated with one item - the creating user's ID. I had to change this in this PR because it was preventing me from making sure the groups exporter was working properly

Reply:
@charliecon Why is this one marked as Do Not Merge.  Are we ready to get this resolved

Reply:
It's marked as "do not merge" because, at the moment, the target branch is main and it is open as a hotfix PR. We were waiting on a response from the forum user before deciding if it should go out as a hotfix or with the regular schedule. Since they still haven't responded, I'll resolve any conflicts and change the target branch to `dev`

Reply:
Made all the updates. This PR is ready to go into dev 👍🏻